superchain/superchain.go

```diff
@@ -76,57 +77,59 @@ const (
 )
 
 type ChainConfig struct {
-	Name         string `yaml:"name"`
-	ChainID      uint64 `yaml:"chain_id"`
-	PublicRPC    string `yaml:"public_rpc"`
-	SequencerRPC string `yaml:"sequencer_rpc"`
-	Explorer     string `yaml:"explorer"`
+	Name         string `yaml:"name" toml:"name"`
+	ChainID      uint64 `yaml:"chain_id" toml:"chain_id"`
+	PublicRPC    string `yaml:"public_rpc" toml:"public_rpc"`
+	SequencerRPC string `yaml:"sequencer_rpc" toml:"sequencer_rpc"`
+	Explorer     string `yaml:"explorer" toml:"explorer"`
 
-	SuperchainLevel SuperchainLevel `yaml:"superchain_level"`
+	SuperchainLevel SuperchainLevel `yaml:"superchain_level" toml:"superchain_level"`
 
 	// If StandardChainCandidate is true, standard chain validation checks will
 	// run on this chain even if it is a frontier chain.
-	StandardChainCandidate bool `yaml:"standard_chain_candidate,omitempty"`
+	StandardChainCandidate bool `yaml:"standard_chain_candidate,omitempty" toml:"standard_chain_candidate,omitempty"`
```

Comment:
Can we remove all yaml struct tags? Is that worth doing, or not? (Genuine Q)

Reply:
Yes we can and should after we're confident the toml config files contain all of the data read from the yaml versions, and we no longer need the yaml tags to help with the one-time conversion (via `add-chain convert-to-toml` temp subcommand)